Driverless AI/natural-language-processing-tutorial-sentiment-analysis /natural-language-processing-tutorial-sentiment-analysis.md

```diff
@@ -526,53 +588,52 @@ There are also several plots adjacent to the summary tab that give insight into
 
 ## Task 6: Custom Recipe to Improve Predictions
 
-The latest version(1.7.0 onwards) of Driverless AI implements a key feature called BYOR[1] which stands for Bring Your Own Recipes. This feature has been designed to enable Data Scientists or domain experts to influence and customize the machine learning optimization used by Driverless AI as per their business needs. This additional feature engineering technique is aimed towards improving the accuracy of the model.
+The latest version(1.7.0) of Driverless AI implements a key feature called BYOR[1] which stands for Bring Your Own Recipes. This feature has been designed to enable Data Scientists or domain experts to influence and customize the machine learning optimization used by Driverless AI as per their business needs. This additional feature engineering technique is aimed towards improving the accuracy of the model.
```

Comment:
@Castro38 should be "The latest version(1.9.0)....."

Reply:
@5675sp Done